radar_control/radar_control.py

```diff
@@ -679,12 +679,19 @@ def radar():
 
                             if beam_time < aveperiod.intt*1e-3:
                                     #TODO maybe log skipping beam?
```

Comment:
I think you can remove this TODO, because I think you are just reducing the time spent on this beam to the time remaining according to the scan bound.  Can also remove the next few lines of commented code

Reply:
I also think it would be nice to rename the beam_time variable to bound_time_remaining, or time_to_next_scanbound or something to make this more clear of what it is. Beams have historically mapped to integration time but not necessarily in the future if we are doing imaging.

Reply:
One more thing, I think lines 209-211 can be removed (no longer a TODO in docs) and also I think line 156 is confusing just due to nomenclature of what a scan is. I think 'A list of seconds past the minute for integration times in a scan to align to.' would be a better doc